tests/datavolume_gc.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	gomegatypes "github.com/onsi/gomega/types"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/tests/util"
+
+	"kubevirt.io/client-go/kubecli"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
+	. "kubevirt.io/kubevirt/tests/framework/matcher"
+)
+
+func DeleteDataVolume(dv *cdiv1.DataVolume) {
+	if dv == nil {
+		return
+	}
+	By(DeletingDataVolume)
+	virtCli, err := kubecli.GetKubevirtClient()
+	util.PanicOnError(err)
+
+	err = virtCli.CdiClient().CdiV1beta1().DataVolumes(dv.Namespace).Delete(context.Background(), dv.Name, metav1.DeleteOptions{})
+	if !IsDataVolumeGC(virtCli) {
+		Expect(err).To(BeNil())
```

Comment:
some places added dv = nil, not sure if should be added here too

Reply:
Adding that. Let's see what @mhenriks thinks about `DeleteDataVolume(dv **v1beta1.DataVolume)`.

Reply:
no dont change it to **, just do dv=nil were it was deleted just after calling this function

Reply:
why not nil them all? I don't see any risk